_pages/security/dynamic-scanning.md

```diff
@@ -5,29 +5,57 @@ parent: Security
 
-1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads).
+1. [Install the ZAP desktop app](https://github.com/zaproxy/zaproxy/wiki/Downloads). A recent version of [Java](http://www.oracle.com/technetwork/java/javase/downloads/index.html) (7 or 8 work) is needed to run ZAP.
 1. In the Quick Start tab, enter the URL of your application.
     * While you will need to run against production for the official ATO, you can also try out ZAP running against a local server.
-        * If you're using Rails on Mac, start it with
-
-            ```bash
-            rails server --binding 0.0.0.0
-            ```
 
 1. Click "Attack".
     * You should see a bunch of URLs start to pop up in the Spider tab, then the Active Scan tab.
     * This will take a while.
-    * There is a progress bar in the Spider and Active Scan tabs. You can click the <img class="inline" src="../../assets/zap_graph.png" alt="small graph icon"/> button in the latter to get more details.
+    * There is a progress bar in the Spider and Active Scan tabs. You can click the <img class="inline" src="../../assets/images/zap_graph.png" alt="small graph icon"/> button in the latter to get more details.
 1. Wait for the "Progress" area to say say "Attack complete".
 1. Open the Alerts tab.
 
 You can now click on each alert to see more details.
 
+### Examining the Results
+
+#### The Spider
+
+![spider results](../../assets/images/spider_results.png)
+As configured, the Quick Start Attack scan does not follow links to other domains or subdomains. If your project uses either (for example, you use S3 for assets, or the api is at a different sub domain), you will want to click <img class="inline" src="../../assets/images/zap_options.png" alt="small options icon"/> and update the options to include the domains & subdomains within the scope.
+
+#### Alerts
+
+![alert results](../../assets/images/alert_results.png)
+The Alerts pane lists all alerts discovered while scanning the site. As described on the [alerts page](../scanning#alerts), the red and orange-flagged alerts must be taken care of before the application can be ATO'd. You have a little more flexibility when dealing with the yellow and blue flags, but all of them must be either corrected or, in the case of false positives, documented.
+
+If you're running the attack against a local server you may see some alerts that you wouldn't see on cloud.gov. Debugging web servers are more 'chatty' about errors than production servers.
+
 ### Authentication
 
+Testing the application in an authenticated state is easiest when done with ZAP acting as a Man In The Middle Proxy. Unfortunately, the "Plug-n-Hack" extension mentioned on the Quick Start page is currently non-functional. By default, ZAP puts an automatic proxy configuration file at http://localhost:8080/proxy.pac. You should set your browser to use that as the proxy for the duration of the testing.
```

Comment:
Can we just link to https://github.com/zaproxy/zap-core-help/wiki/HelpStartProxies instead?